pages/learn/develop/runtime.md

```diff
@@ -80,6 +80,8 @@ services:
       io.balena.features.dbus: '1'
 ```
 
+__Note:__ Please be aware that setting `DBUS_SYSTEM_BUS_ADDRESS` as a service or environment variable and enabling systemd at the same time might introduce unexpected side effects. systemd might interact with the host system instead of the container. These interactions can potentially cause balenaOS devices to disconnect from balenaCloud or even fall offline. So we advise our users to prefer the prepending method.
```

Comment:
```suggestion
__Note:__ Please be aware that setting `DBUS_SYSTEM_BUS_ADDRESS` as a service or environment variable and enabling `systemd` at the same time might introduce unexpected side effects. `Systemd` might start to interact with the host system instead of the container. These interactions can potentially cause balenaOS devices to disconnect from balenaCloud or even go offline. Hence, users are advised to use the `io.balena.features.dbus` label instead.
```
Always being 3rd person and keeping things specific helps a ton in the docs.

Reply:
Funny that you suggest these @vipulgupta2048 as I also thought about these working on this PR. I didn't commit as you suggested initially for consistency reasons. Right now the docs always refer it as systemd and not `systemd`. Similarly, I saw mixed usage on how we refer to the users. 

I'm fine with committing your suggestion but I wanted to highlight the potential inconsistency we create, especially with `systemd` reference.

Reply:
On another note
> users are advised to use the `io.balena.features.dbus` label instead.

Not sure if I follow. How do you reason this? 

Admittedly, I don't know much about DBUS but in the linked support threads the agents kept re-iterating: "User was advised not to set DBUS_SYSTEM_BUS_ADDRESS as a container-wide setting and instead to put the variable definition before executing the process on the same line".

The way I see the `io.balena.features.dbus` label needs to be set regardless. It's more about warning the users on the potential side effects of exporting `DBUS_SYSTEM_BUS_ADDRESS` as an environment variable.

Reply:
> I'm fine with committing your suggestion but I wanted to highlight the potential inconsistency we create, especially with systemd reference.

I agree, consistency is important. If that's something we follow all over. We shouldn't change it.

Reply:
> The way I see the io.balena.features.dbus label needs to be set regardless. It's more about warning the users on the potential side effects of exporting DBUS_SYSTEM_BUS_ADDRESS as an environment variable.

Right on. I made this change in regards to this sentence here where I think being specific to what that pretending method is makes more sense. 

> So we advise our users to prefer the prepending method.

Reply:
We can replace what I suggested with what exactly the prepending method is that the customers should use.

Reply:
Cool! I pushed a new change that combines all we discussed so far Vipul. Please take another look.